src/Tools/Auditor/assets/template.html

```diff
@@ -0,0 +1,526 @@
+<!DOCTYPE html>
```

Comment:
Let's patch out all the vendored code and refer to it by URL to avoid license complications and repo bloat